src/compiler/bundle/ext-transforms-plugin.ts

```diff
@@ -56,6 +80,22 @@ export const extTransformsPlugin = (
           return null;
         }
 
+        /**
+         * add file to watch list if it is outside of the `srcDir` config path
+         */
+        if (config.watch && (id.startsWith('/') || id.startsWith('.')) && !id.startsWith(config.srcDir)) {
+          compilerCtx.addWatchFile(id.split('?')[0]);
+        }
```

Comment:
This can't happen since there is output for styles, right? All styles are inlined with the component logic in a JS file and the transform is only triggered on css/sass files. Are there output targets that define external CSS files instead of inlining them?